packages/mds-metrics/api.ts

```diff
@@ -18,7 +18,13 @@ import express from 'express'
 
 import { pathsFor } from '@mds-core/mds-utils'
 import { checkAccess } from '@mds-core/mds-api-server'
-import { getStateSnapshot, getEventSnapshot, getTelemetryCounts, getEventCounts } from './request-handlers'
+import {
+  getStateSnapshot,
+  getEventSnapshot,
+  getTelemetryCounts,
+  getEventCounts,
+  getDumpMetrics
```

Comment:
rename to getMetricsDump ?

Reply:
With the rename proposed below, should just be `getAll`